src/main/java/com/oitsjustjose/geolosys/client/render/GeolosysClient.java

```diff
@@ -0,0 +1,30 @@
+package com.oitsjustjose.geolosys.client.render;
+
+import com.oitsjustjose.geolosys.common.config.ClientConfig;
+import net.minecraft.client.Minecraft;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.player.ItemTooltipEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class GeolosysClient {
+    public static void setup() {
+        MinecraftForge.EVENT_BUS.register(GeolosysClient.class);
+        MinecraftForge.EVENT_BUS.register(ProPickOverlay.class);
+    }
+
+    @SubscribeEvent
+    public void onHover(ItemTooltipEvent event) {
```

Comment:
I haven't tested this locally yet, but just looking - does this need to be static since you're registering the class, and not an instance, of `GeolosysClient` to the Event Bus?

Reply:
Probably. Let me fix that quickly

Reply:
Should be fixed now